example/src/main/java/Main.java

```diff
@@ -70,6 +70,7 @@ private void initFonts(final ImGuiIO io) {
 
     @Override
     public void process() {
+        ImGui.setWindowSize(400, 410);
```

Comment:
Didn't find where the Demo's window height is set 🤔
But I think that we can change the window's size by doing this

Reply:
You'll change a size of the next window, not the window with "extra" checkboxes. Wrap everything to their own window and change it.

Reply:
Done.
Also added brackets to an if statement that I forgot.